packages/backend/src/controllers/authentication.ts

```diff
@@ -138,18 +137,12 @@ export const googlePassportStrategy: GoogleStrategy | undefined = !(
                       openIdUser,
                       req.user,
                   );
-                  if (isSessionUser(user)) {
-                      return done(null, user);
-                  }
+                  return done(null, user);
+              } catch (e) {
+                  Logger.warn(`Failed to authorize user. ${e}`);
                   return done(null, false, {
-                      message: 'Cannot find existing Lightdash account',
+                      message: 'Failed to authorize user',
                   });
```

Comment:
@owlas When we were defining the error it wasn't redirecting to the login page. I'm also keeping the error message generic for security.